solution/backend/regcore/search/models.py

```diff
@@ -22,12 +22,11 @@ def search(self, query):
             cover_density = True
         search_query = SearchQuery(query, search_type=search_type, config='english')
 
-        return self.annotate(vector_column=RawSQL("vector_column", [], output_field=SearchVectorField()))\
-            .annotate(rank=SearchRank(
-                "vector_column",
+        return self.annotate(rank=SearchRank(
+                RawSQL("vector_column", [], output_field=SearchVectorField()),
                 search_query, cover_density=cover_density)
             )\
-            .filter(rank__gte=0.04)\
+            .filter(rank__gte=0.2)\
```

Comment:
This adjustment got ranking to be higher, to around what it was recently so increasing the threshold of rank makes sense.  Before it got really low for some reason.(like scientific notation low in some cases)